modules/pruning-images.adoc

```diff
@@ -55,29 +49,28 @@ status:
     reason: Succeeded
     message: "Most recent image pruning job succeeded."
 ----
-<1> `schedule`: `CronJob` formatted schedule, defaults to daily at midnight for new clusters. This is a required field.
-<2> `suspend`: If set to `true`, the `CronJob` running pruning is suspended. This is an optional field, and it defaults to `false`.
-<3> `keepTagRevisions`: The number of revisions per tag to keep. This is an optional field, and it defaults to `3` if not set.
-<4> `keepYoungerThan`: Retain images younger than this duration. This is an optional field, and it defaults `60m` if not set.
+<1> `schedule`: `CronJob` formatted schedule. This is an optional field, default is daily at midnight.
+<2> `suspend`: If set to `true`, the `CronJob` running pruning is suspended. This is an optional field, default is `false`. The initial value on new clusters is `false`.
```

Comment:
This differs from 4.4. On 4.4 the pruner config is bootstrapped with `suspend: true`.